app/components/navigation/navigation.tsx

```diff
@@ -26,36 +27,40 @@ export default function Navigation({ pages }: {
-            const subpageInputAttr = {
-                id: `subpage-${keyName}-input`,
-                classes: styles["nav-subpages-toggle"],
-            };
-            const subpageLabelAttr = {
-                id: `subpage-${keyName}-label`,
-                content: <p className={styles["top-level-link"]}>{page.name}</p>,
-                classes: styles["nav-subpages-menu"],
-                aria: {
-                    controls: `subpage-${keyName}-dropdown`
-                },
-                testId: `subpage-${keyName}-label`
-            };
-            const subpageListAttr = {
-                id: `subpage-${keyName}-dropdown`,
-                testId: `subpage-${keyName}-dropdown`,
-                classes: styles["nav-subpages-list"]
-            };
-            return (<li key={`nav-toplevellink-${keyName}`} className={styles["nav-toplevel-page"]}>
-                <NavigationItem hasSubpages={hasSubpages }  inputAttr={subpageInputAttr } labelAttr={subpageLabelAttr } listAttr={ subpageListAttr} childElements={page.subpages.map((subpage: ISubpage) => {
-                    const subpageKeyName = subpage.name.toLowerCase().replaceAll(" ", "-");
-                    return <li key={`nav-toplevellink-${keyName}-sublink-${subpageKeyName}`}>
-                        <Link href={subpage.path} className={styles["sub-level-link"]}>{subpage.name}</Link>
-                    </li>;
-                })} />
-            </li>);
-        })} />
+        <NavigationItem inputAttr={rootNavInputAttr} labelAttr={rootNavLabelAttr} listAttr={rootNavListAttr} hasSubpages={ pages.length > 0} childElements={<>{
+            pages.map((page: IPage) => {
+                const keyName = page.name.toLowerCase().replaceAll(" ", "-");
+                const hasSubpages = page.subpages && page.subpages.length > 0;
+                const subpageInputAttr = {
+                    id: `subpage-${keyName}-input`,
+                    classes: styles["nav-subpages-toggle"],
+                };
+                const subpageLabelAttr = {
+                    id: `subpage-${keyName}-label`,
+                    content: <p className={`${styles["top-level-link"]} ${styles["nav-dropdown"]}`}>{page.name}</p>,
+                    classes: styles["nav-subpages-menu"],
+                    aria: {
+                        controls: `subpage-${keyName}-dropdown`
+                    },
+                    testId: `subpage-${keyName}-label`
+                };
+                const subpageListAttr = {
+                    id: `subpage-${keyName}-dropdown`,
+                    testId: `subpage-${keyName}-dropdown`,
+                    classes: styles["nav-subpages-list"]
+                };
+                return (<li key={`nav-toplevellink-${keyName}`}>
+                    <NavigationItem hasSubpages={hasSubpages }  inputAttr={subpageInputAttr } labelAttr={subpageLabelAttr } listAttr={ subpageListAttr} childElements={page.subpages.map((subpage: ISubpage) => {
```

Comment:
nit: lots of extra spaces... 😬 
```suggestion
                    <NavigationItem hasSubpages={hasSubpages}  inputAttr={subpageInputAttr} labelAttr={subpageLabelAttr} listAttr={subpageListAttr} childElements={page.subpages.map((subpage: ISubpage) => {
```

Reply:
Will clean up :)